go/chat/msgchecker/constants.go

```diff
@@ -8,20 +8,22 @@ const (
 	ReactionMessageMaxLength = 50
 	HeadlineMaxLength        = 280
 	TopicMaxLength           = 20
+	PaymentTextMaxLength     = 240
```

Comment:
Do we want to keep this? It's for checking ``Requestpayment.Note` length. We can use `TextMessageMaxLength` for that.

Reply:
Seems fine to keep it.